base/logging.go

```diff
@@ -803,6 +803,28 @@ var (
 	debugLogger, infoLogger, warnLogger, errorLogger *FileLogger
 )
 
+// RotateLogfiles rotates all active log files.
+func RotateLogfiles() error {
+	Infof(KeyAll, "Rotating log files...")
+
+	// Even if one errors, we still want to try and rotate the rest, hence the weird error handling.
+	var err error
```

Comment:
Could this be a slice of `FileLoggers` that you could then iterate over the slice and call `Rotate()` on each one?  

Also, it seems odd to keep clobbering the error .. I would think if there is one error, maybe abort early

Reply:
@tleyden what do you think of the new approach I've just pushed up?